File: 02_Lenguajes-de-programación/03_C#/02_operadores.cs
using System;

class Program {
  public static void Main (string[] args) {

    /* Operadores */

    // 1. Operadores aritméticos
    Console.WriteLine(1 + 1); // Resultado: 2
    Console.WriteLine(1 - 1); // Resultado: 0
    Console.WriteLine(1 * 1); // Resultado: 1
    Console.WriteLine(1 / 1); // Resultado: 1
    Console.WriteLine(1 % 1); // Resultado: 0

    // 2. Operadores comparativos
    //Console.WriteLine(1 == "1"); // Da un error al comparar un entero con un texto, porque no son del mismo tipo.
    Console.WriteLine(1 == 1); // Resultado: True
    Console.WriteLine(1 != 1); // Respuesta: False
    Console.WriteLine(1 < 1); // Respuesta: False
    Console.WriteLine(1 > 1); // Respuesta: False
    Console.WriteLine(1 <= 1); // Respuesta: True
    Console.WriteLine(1 >= 1); // Respuesta: True

    // 3. Operadores lógicos
    Console.WriteLine(true && true); // Respuesta: True
    Console.WriteLine(true && false); // Respuesta: False
    Console.WriteLine(false && true); // Respuesta: False
    Console.WriteLine(false && false); // Respuesta: False

    Console.WriteLine(true || true); // Respuesta: True
    Console.WriteLine(true || false); // Respuesta: True
    Console.WriteLine(false || true); // Respuesta: True
    Console.WriteLine(false || false); // Respuesta: False

    Console.WriteLine(!true); // Respuesta: False
    Console.WriteLine(!false); // Respuesta: True
  }
}

File: 02_Lenguajes-de-programación/08_C-C++/02_Sintaxis_C++/02_operadores.cpp
#include <iostream>

int main() {

    int a = 10;
    int b = 20;

    bool a = true;
    bool b = false;

    // Operadores Logicos
    std::cout << "a AND b: " << (a && b) << std::endl;
    std::cout << "a OR b: " << (a || b) << std::endl;
    std::cout << "NOT a: " << (!a) << std::endl;

    // Operadores Comparativos
    std::cout << "a == b: " << (a == b) << std::endl;
    std::cout << "a != b: " << (a != b) << std::endl;
    std::cout << "a > b: " << (a > b) << std::endl;
    std::cout << "a < b: " << (a < b) << std::endl;
    std::cout << "a >= b: " << (a >= b) << std::endl;
    std::cout << "a <= b: " << (a <= b) << std::endl;

    // Operadores Aritméticos
    std::cout << "a + b: " << (a + b) << std::endl;
    std::cout << "a - b: " << (a - b) << std::endl;
    std::cout << "a * b: " << (a * b) << std::endl;
    std::cout << "a / b: " << (a / b) << std::endl;
    std::cout << "a % b: " << (a % b) << std::endl;

    return 0;

}
